pkg/kn/commands/logger/logger.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2023 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logger
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+type Config struct {
+	QuietMode bool
+}
```

Comment:
`Config` struct should accept io.Writer as well. There might be other use cases that benefit from output redirect, not always being `os.Stdout`. Then it should be initialized with value of: 

```
cmd.OutOrStdout()
```

Reply:
That's the case for failing unit tests, because output is redirected and captured to assert.

Reply:
IMO this still seems like a problematic part, at least for unit tests. @sreeram-venkitesh pls let me know if you can take a look at it. Thanks!